jena-arq/src/main/java/org/apache/jena/riot/system/Checker.java

```diff
@@ -40,19 +40,30 @@
 /**
  * Functions for checking nodes, triples and quads.
  * <p>
- * If the errorHandler is null, use the system wide handler.
+ * The check functions have two basic signatures:<br>
+ * 1. "<tt>check...(<i>object</i>)</tt>"
+ * 2. "<tt>check...(<i>object, errorHandler, line, col</i>)</tt>"
+ * The first type are for boolean testing and do not generate output. They call the
+ * second type with default values for the last 3 parameters: nullErrorHandler, -1L, -1
+ * The second type are for boolean testing and optionally generate error handling
+ * output.
  * <p>
- * If the errorHandler line/columns numbers are -1, -1, messages do not include them.
+ * Parameters:<br>
+ * {@link ErrorHandler} errorHandler - the errorHandler for output. If the errorHandler
+ * is null, use the system wide handler.<br>
+ * {@link long} line - code line number generating the check.<br>
+ * {@link long} col - code column number generating the check.<br>
+ * If the errorHandler is null, the line and column numbers not used.
  * <p>
- * Operations "<tt>checkXXX(<i>item</i>)</tt>" are for boolean testing
- * and do not generate output.
+ * The functions "<tt>check...(<i>object</i>)</tt>" are for boolean testing and do not
+ * generate output.
  */
 
 public class Checker {
 
     /** A node -- must be concrete node or a variable. */
     public static boolean check(Node node) {
-        return check(node, nullErrorHandler, -1, -1);
+        return check(node, nullErrorHandler, -1L, -1L);
```

Comment:
Side note: `-1` is converted by the compiler to `-1L` to match the signature so this is zero runtime cost.
Leaving them changed to `-1L` is OK.

Reply:
Yeah, I thought that, since it was explicit in some places, just make them consistent.